src/library_webgl.js

```diff
@@ -4,13 +4,31 @@
  * SPDX-License-Identifier: MIT
  */
 
+// Specifies the size of the GL temp buffer pool, in bytes. Must be a multiple
+// of 9 and 16.
+{{{ GL_POOL_TEMP_BUFFERS_SIZE = 2*9*16 }}} // = 288
+
 var LibraryGL = {
 
   // For functions such as glDrawBuffers, glInvalidateFramebuffer and glInvalidateSubFramebuffer that need to pass a short array to the WebGL API,
   // create a set of short fixed-length arrays to avoid having to generate any garbage when calling those functions.
-  _tempFixedLengthArray__postset: 'for (var i = 0; i < 32; i++) __tempFixedLengthArray.push(new Array(i));',
+  _tempFixedLengthArray__postset: 'for (var i = 0; i < 32; ++i) __tempFixedLengthArray.push(new Array(i));',
   _tempFixedLengthArray: [],
 
+  _miniTempFloatBuffers: [],
+  _miniTempFloatBuffers__postset:
```

Comment:
I still think it would be good to have `GL` in the name of these. it helps when reading the generated code to know what they are immediately, since they are in the global scope with a lot of other stuff.

Reply:
check